src/js/api/core-shim.js

```diff
@@ -226,15 +226,17 @@ Object.assign(CoreShim.prototype, {
 
 function setupError(core, error) {
     let message = error.message;
-    if (message.indexOf(':') === -1) {
-        message = `Error loading player: ${message}`;
+    const index = message.indexOf(':');
+    let title = 'Error loading player:';
+    if (index > 0) {
+        title = error.message.substr(0, index + 1);
+        message = error.message.substr(index + 1);
```

Comment:
Improve readability by ordering variable declaration, reuse `message` and only add `1` once:

```js
let title = 'Error loading player:';
let message = error.message;
const index = message.indexOf(':') + 1;
if (index > 0) {
    title = message.substr(0, index);
    message = message.substr(index);
```